android/src/main/java/org/conscrypt/Platform.java

```diff
@@ -969,4 +969,8 @@ private static void writeStats(
     public static boolean isJavaxCertificateSupported() {
         return true;
     }
+
+    public static boolean isTlsV1Deprecated() {
+        return false;
```

Comment:
I think in this PR we can set these to true, and we can announce it as part of the next public release - I don't think we need the same degree of messaging as the Android platform.

Reply:
It appears I have done this in a separate cl is that ok?

Reply:
Yup, that's exactly the way to do it.  When the PR is submitted there's a "squash and merge" option we use so that it only ends up as a single commit on the main branch.

Thanks!